frontend/src/components/GeyserUnstakeView.tsx

```diff
@@ -0,0 +1,72 @@
+import { BigNumber, BigNumberish, Wallet } from 'ethers'
```

Comment:
since these views are mostly the same, we prolly should simplify this a little. 

potential approach: 

- there should only be one geyser specific view and the mode can be stored in the context -- maybe the `geysercontext`?. 
- this `mode` determines what button gets showed on the geyser specific view. 
- The button can be separated out into its own component -- depending on the mode it knows what function to attach to its `onClick` and what text to display (or this could also be done in the geyser specific view and pass the info along into props simplifying the button)

Reply:
I left the two views as two separate components because the design was not complete, and I was expecting the two views to be quite different apart from the input and button. We can look into simplifying it when we get the final designs and take some pieces out that will be reused in both views?

Reply:
we can optimize and refactor later as well, def not high prio. lets leave a TODO somewhere in the file since it doesn't look like the stake/unstake view will change too much (the only difference prolly just being the button)

Reply:
added TODO comment